src/host/ncp_spinel.cpp

```diff
@@ -676,6 +686,15 @@ void NcpSpinel::HandleValueInserted(spinel_prop_key_t aKey, const uint8_t *aBuff
         break;
     }
 #endif // OTBR_ENABLE_SRP_ADVERTISING_PROXY
+    case SPINEL_PROP_BACKBONE_ROUTER_MULTICAST_LISTENER:
+    {
+        const otIp6Address *addr;
+
+        VerifyOrExit(decoder.ReadIp6Address(addr) == OT_ERROR_NONE, error = OTBR_ERROR_PARSE);
```

Comment:
nit: `SuccessOrExit`

Reply:
It's a little tricky here. Because the type of error is 'otbrError'. So I used `VerifyOrExit` here to convert the otError to otbrError flexibly.